src/app/content/types.ts

```diff
@@ -132,3 +132,6 @@ export interface ArchivePage {
 }
 
 export type ArchiveContent = ArchivePage | ArchiveBook;
+
+// tslint:disable-next-line: max-line-length
+export type SearchButtonColor = 'blue' | 'green' | 'gray' | 'yellow' | 'deep-green' | 'light-blue' | 'orange' | 'red' | 'gray';
```

Comment:
is this the same as the theme colors? is it the theme colors plus gray? you have gray in there twice. can you derive it from the theme colors?

also this could go under content/search